File: POOUber/PHP/uberSUV.php
<?php
require_once('carAdvanced.php');

class UberSUV extends CardAdvanced {

    public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial){

        $this->license = $license;
        $this->driver = $driver;
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;

    }

}

?>

File: POOUber/Java/UberSUV.java
import java.util.ArrayList;
import java.util.Map;

class UberSUV extends CarAdvanced {    
    
    protected Integer passenger;

    public UberSUV(String license, Account driver, Map<String, Map<String, Integer>> typeCarAccepted,
            ArrayList<String> seatsMaterial) {

        super(license, driver, typeCarAccepted, seatsMaterial);

    }

    @Override
    public void setPassenger(Integer passenger) {

        if (passenger == 6) {

            this.passenger = passenger;

        }

    }

}
